Doc/library/stdtypes.rst

```diff
@@ -2518,8 +2518,26 @@ objects.
       >>> bytearray(b'\xf0\xf1\xf2').hex()
       'f0f1f2'
 
+      If you want to make the hex string easier to read, you can specify a
```

Comment:
This was already documented at https://github.com/python/cpython/pull/13578/files#diff-7498e907ba97646df434a0eb583c6909R2407 to be new in Python 3.8 and looks like a duplicate addition.

Reply:
The case is that `bytes.hex(sep=..., bytes_per_sep=...)` is already documented to be new in 3.8, but `bytearray.hex(sep=..., bytes_per_sep=...)` is not (the `versionchanged:: 3.8` indicator is missing). The "If you want to make the hex string easier to read" paragraph is actually copied from the document of `bytes.hex` (just in the PR you mentioned).

Reply:
And `memoryview.hex(sep=..., bytes_per_sep=...)` also need to be marked as new in 3.8.

Reply:
I feel it's better to add the `versionchanged` directive to bytearray and memoryview but not sure about duplicating the examples per section. Adding initial docs author for review.

Reply:
thanks!  this looks good, i like that you just link back to bytes.hex for the detailed description.